src/znc.cpp

```diff
@@ -628,6 +629,7 @@ bool CZNC::WriteNewConfig(const CString& sConfigFile) {
     VCString vsLines;
 
     vsLines.push_back(MakeConfigHeader());
+    vsLines.push_back("PidFile = /run/znc/znc.pid");
```

Comment:
There is another use case, even on Gentoo: to not use the provided init scripts, but just run `znc --makeconf` as your own user.

> If you are using a runscript/unit file you need a known resource you can check for status. I.e. if znc is running there must be a pidfile and this pid must be alive (just checking for znc processes isn't enough when you want to support multiple instances).

With `start-stop-daemon --background --make-pidfile` you can run `znc --foreground`, and then start-stop-daemon will put ZNC to background itself, and write its pidfile.